package.json

```diff
@@ -77,5 +77,8 @@
     "lint:style",
     "lint:js"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {
+    "@ember/edition-utils": "^1.1.1"
```

Comment:
All app dependencies are dev dependencies. It doesn't really matter too much, but it's the ember-cli pattern.

Reply:
fixed.